pilot/pkg/kube/inject/webhook.go

```diff
@@ -241,6 +247,13 @@ func (wh *Webhook) Run(stop <-chan struct{}) {
 	}
 }
 
+// GetCABundlePEM returns the CA bundle PEM, read from the cert file. Safe for concurrent use.
```

Comment:
Is the intent here to support CA cert rotation? If not, the caCertPEM re-load code can be removed.

Reply:
Yes, might as well include this since it's a minor addition.